snippets/ztp-example-siteconfig.adoc

```diff
@@ -16,7 +16,10 @@ spec:
   clusters:
   - clusterName: "<site_name>"
     networkType: "OVNKubernetes"
-    clusterLabels: <4>
+    extraManifests: <4>
+      searchPaths: <5>
```

Comment:
here searchPaths is marked at <5>, but in the description <5> represents clusterLabels

Reply:
@sabbir-47 
Thanks, I've corrected it.